components/prism-v.js

```diff
@@ -0,0 +1,73 @@
+Prism.languages.v = Prism.languages.extend('clike', {
+	'string': [
+		{
+			pattern: /`(?:\\\`|\\?[^\`]{1,2})`/, // using {1,2} instead of `u` flag for compatibility
+			alias: 'rune'
+		},
+		{
+			pattern: /r?(["'])(?:\\(?:\r\n|[\s\S])|(?!\1)[^\\\r\n])*\1/,
+			alias: 'quoted-string',
+			greedy: true,
+			inside: {
+				'interpolation': {
+					pattern: /((?:^|[^\\])(?:\\{2})*)\$(?:\{[^{}]*\}|\w+(?:\.\w+(?:\([^\(\)]*\))?|\[[^\[\]]+\])*)/,
+					lookbehind: true,
+					inside: {
+						'interpolation-variable': {
+							pattern: /^\$\w[\s\S]*$/,
+							alias: 'variable'
+						},
+						'interpolation-punctuation': {
+							pattern: /^\${|}$/,
+							alias: 'punctuation'
+						},
+						'interpolation-expression': {
+							pattern: /[\s\S]+/,
+							inside: Prism.languages.v
```

Comment:
I'm sorry that I suggested code that doesn't work. `Prism.languages.v` isn't defined yet, so `inside` will be set to `undefined`.

To fix this, please replace this line with:
```suggestion
							inside: null // see below
```

And add the following line after the `Prism.languages.extend` call.

```js
Prism.languages.v['string'][1].inside['interpolation'].inside['interpolation-expression'].inside = Prism.languages.v;
```